git/README.md

```diff
@@ -29,9 +29,9 @@ git checkout -b <branch-name>
 
 Prefix the branch name with something meaningful, for example:
```

Comment:
It would be nice to provide some sort of explanation for the slashes, not just _what_ we do but _why_.

Reply:
@tylersticka PR updated. Does that work better? :)

Reply:
It does indeed! 👍